get_started.md

```diff
@@ -2,42 +2,59 @@
+
+You can also control time! Click the fast-forward icon by **Location at time** to set the time, which is important for visitors like Jane who cannot enter a room until a certain time.
 
 ### Ruleset, DDL, and Output tabs
 
 There are three tabs on the left. These are windows that show:
 
-- Ruleset: Displays the Declarative C++ code that handles TODO. You can modify the code in this tab and rebuild the sample to see how the changes affect the way that the sample runs. Rulesets encode the "business logic" that TODO
-- DDL: Displays the DDL code that describes the layout of the TODO, the state of the TODO, and events the factory produces.  In short, the DDL represents the data model for the app.
-- Output: Displays details when building/running your program.
+- **Ruleset:** Declarative C++ code that handles all the "business logic" of building and room access. You can edit the code in this tab and rebuild the sample to see how it affects the way that the sample runs.
+- **DDL:** A database schema describing the "data model" of everything in this Access Control scenario.
+- **Output:** Displays details while building/running your program.
 
 ## Build and run the sample
 
-For the best user experience, run the Sandbox in full screen mode set to Full HD or higher resolution.
+For the best user experience, keep the Sandbox's browser maximized to full-screen mode and use 1080p or higher resolution.
 
 The Sandbox contains a complete sample for you to examine and modify.
 
 To build the sample, select the **Build** button located above the simulator window. The state of the button changes depending on what point in the build/run cycle that you are in.
 
-While the sample builds, the **Output** window displays the progress of the build.
-
-**Note**: It can take several minutes to build the sample. 
+While the sample builds, the **Output** window displays the progress of the build. **It can take several minutes** to build the sample.
 
-When the sample has completed building, select the **Run** button. 
+When the sample has completed building, press **Run**.
 
-TODO
+### Example interaction
+1. Build and run the sample.
+2. Click John's **Badge** icon to scan his badge at the HQ building.
+3. Click John's **Face** icon to scan his face at the HQ building and let him in.
+4. Click Jane's **Badge** then **Face** icon. She will not be allowed in!
+5. Click the **Fast-forward** button until it reads `10:00 am`.
+6. Again, click Jane's **Badge** then **Face** icon and she will be let into the HQ building!
+7. Change John and Jane's drop-down boxes to say `Big Room` instead of `Little Room`.
+8. Click John and Jane's **Face** icons to scan them into the Big Room for their "boring meeting".
+9. Try other interactions!
```

Comment:
Make this a non-numbered item.

Now, try other interactions to see how the rules interact.